README.md

```diff
@@ -1,86 +1,130 @@
 
-### Docker
-The Docker container is set up in such a way that the static-server will serve the example-client.
+*Note: Enzyme is [not ready for React 17
+yet](https://github.com/enzymejs/enzyme/issues/2429), so we're using
+an unofficial version and installing packages with `npm i
+--legacy-peer-deps` if using node v7.  We will change to an official
+version of Enzyme when a compatible released version is available.*
 
-To the example client and static server in docker run the following commands:
+## Deployment
 
-- `docker build --build-arg REACT_APP_UPLOAD_HOST="" --build-arg REACT_APP_CHAIN_ID={REACT_APP_CHAIN_ID_VALUE} --build-arg  REACT_APP_CHAIN_NAME={REACT_APP_CHAIN_NAME_VAUE}  --build-arg REACT_APP_CHAIN_HOST={REACT_APP_CHAIN_HOST} --build-arg REACT_APP_TORUS_BUILD_ENV={REACT_APP_TORUS_BUILD_ENV_VALUE} . -t example-client`
-- `docker run  --init --rm  -p 8080:8080 -v {name_for_volume}:/app/static-server/public example-client`
+The following steps should get you up and running quickly (assuming a
+working knowledge of NPM and Git):
 
-#### Docker Builds
-Docker image will be pushed from our [github repo](https://github.com/LibertyDSNP/example-client) to our [dockerhub](https://hub.docker.com/r/dsnp/example-client) when a new
-git tag is created.
+* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
 
-Tags will have the following format - `docker/*` i.e. `docker/v1.0.0`.
+* Install the correct npm and node version.  We recommend using asdf: `asdf install`
 
-***
-## Project Setup
-Follow this quick start guide to get up and running. This guide assumes a working knowledge of npm and git. Make sure to check out [CONTRIBUTING.md](CONTRIBUTING.md) as well.
-* Clone the example-client repository: `git clone git@github.com:LibertyDSNP/example-client.git`
-* Install the correct npm and node version. Recommend using asdf: `asdf install`
-* Install modules: `npm install`
+* Install the modules: `npm install`
   * Install Static Server modules `cd ./static-server && npm install`
+
 * Copy `.env.example` to `.env` and edit as needed.
-* Start the Static Server `cd ./static-server && npm run start`
+
+* Start the Static Server: `cd ./static-server && npm run start`
+
 * Start up site locally: `npm run start`
-* Visit http://localhost:3000 (or you configured port)
+
+* Visit http://localhost:3000 (or whatever your configured port is)
+
+You can run the test suite with 'npm run test'.
+
+### Docker-based deployment
+
+The Docker container is set up in such a way that the Static Server
+will serve the Example Client.  To launch it, run the following
+commands:
+
+* `docker build --build-arg REACT_APP_UPLOAD_HOST="" --build-arg REACT_APP_CHAIN_ID={REACT_APP_CHAIN_ID_VALUE} --build-arg  REACT_APP_CHAIN_NAME={REACT_APP_CHAIN_NAME_VALUE}  --build-arg REACT_APP_CHAIN_HOST={REACT_APP_CHAIN_HOST} --build-arg REACT_APP_TORUS_BUILD_ENV={REACT_APP_TORUS_BUILD_ENV_VALUE} . -t example-client`
```

Comment:
This needs documentation about where all these environment variables are set and what they should be.

Reply:
Thanks, @rabble.  I agree.  I only didn't try to add that documentation because I'm not very experienced in either Docker or React, so I wasn't sure what knowledge should be assumed on the part of the intended audience and what shouldn't.  However, I'd love to have some more information about those environment variables in that section.

Reply:
Let's save this for a different PR.